components/Header/components/language-switcher.tsx

```diff
@@ -78,7 +78,16 @@ export const LanguageSwitcher: React.FC = () => {
       }))}
       renderOption={renderSelectOption}
       style={{ width: 110 }}
-      comboboxProps={{ width: 180, position: "bottom-start" }}
+      comboboxProps={{
+        width: 180,
+        position: "bottom-end",
+        withinPortal: true,
+        offset: 5,
+        middlewares: {
+          flip: true,
+          shift: true,
+        },
+      }}
```

Comment:
Prompt:
When I have dropdown - for example with the langauges, it expands to the right side and moves the site. Can we control and check what is changed and how we can fix this?